Persistant-Storage-Snapshot/DeserializationEx.java

```diff
@@ -0,0 +1,35 @@
+// where should this go? same folder as ServerMain? as methods in Snapshotter or ServerMain?
```

Comment:
Actually it looks like the Model is created in `Server`:
https://github.com/lianasyrkett/CodeU-Summer-2017/blob/da1993fab6e3aebd4d93e3ceb6bd46359d4dbd2b/src/codeu/chat/server/Server.java#L61